blog/docs/steering/2021-end-user-election.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2021 Steering Committee End User Seat Election
+date: 2021-12-17
+---
+
+Hello, Knative community,
+
+We want to kick off the Knative Steering Committee (SC) End User election, and we are ready to accept nominations from End Users!
+
+This is the first time an End User is elected to become a member of the Knative Steering Committee (SC)
+
+The Knative Steering Committee (SC) is responsible for the general health of the Knative community. The End User seat will bring more diversity into the SC representation, which the SC will elect.
```

Comment:
Well, this belies my comment above, "Since you spell out Steering Committee (SC) everywhere else". In this case I still think it's probably fine to just spell it out here as well. I guess overall it's just not really clear why it would only be shortened in one or two places, but spelled out everywhere else.
```suggestion
The Knative Steering Committee is responsible for the general health of the Knative community. The End User seat will bring more diversity into the Steering Committee representation, which the Steering Committee will elect.
```